petclinic-frontend/src/features/inventories/InventoryProducts.tsx

```diff
@@ -309,6 +309,12 @@ const InventoryProducts: React.FC = () => {
       ) : (
         <p>No products found.</p>
       )}
+      <button
```

Comment:
Sorry for inadvertently removing your form from the front-end, otherwise great feature. Makes it neat